special-pages/pages/new-tab/app/components/App.module.css

```diff
@@ -34,40 +34,85 @@ body:has([data-reset-layout="true"]) .tube {
 
 .layout {
     display: grid;
-    grid-template-columns: auto 0;
-    transition: all ease .3s;
+    grid-template-columns: auto 0 0;
+    grid-template-areas: "main gap aside";
+    transition: all .3s ease-in-out;
     position: relative;
+    z-index: 1;
 
     &[data-drawer-visibility='visible'] {
-        grid-template-columns: auto var(--ntp-drawer-width);
+        grid-template-columns: auto 4px var(--ntp-combined-width);
     }
 }
 
 .main {
-    &[data-customizer-kind="drawer"] {
-        overflow: hidden;
-        height: 100vh;
+    height: 100vh;
+    overflow: auto;
+    grid-area: main;
+    color: var(--ntp-text-normal);
+}
```

Comment:
A big part of this PR, is un-blocking the layout for the drawer, even though windows doesn't use it yet. So the main content area now behaves as an independent scroll whether the drawer is there or not.